README.md

```diff
@@ -1,18 +1,29 @@
-[![LGTM](https://img.shields.io/lgtm/alerts/github/ampproject/amphtml.svg)](https://lgtm.com/projects/g/ampproject/amphtml/)
-[![Percy](https://percy.io/static/images/percy-badge.svg)](https://percy.io/ampproject/amphtml)
-[![Renovate](https://img.shields.io/badge/renovate-enabled-brightgreen.svg)](https://renovateapp.com/)
-[![Prettier](https://img.shields.io/badge/code_style-prettier-ff69b4.svg?style=flat-square)](https://github.com/prettier/prettier)
+##### Stats:
+
+[![Build Status](https://img.shields.io/travis/ampproject/amphtml/master.svg?logo=Travis%20CI&logoColor=white&style=flat-square "Build Status")](https://travis-ci.org/ampproject/amphtml/builds)
+[![GitHub Release](https://img.shields.io/github/release/ampproject/amphtml.svg?logo=GitHub&style=flat-square "GitHub Release")](https://github.com/ampproject/amphtml/releases)
+[![Contributors](https://img.shields.io/github/contributors-anon/ampproject/amphtml.svg?logo=GitHub&style=flat-square "Contributors")](https://github.com/ampproject/amphtml/graphs/contributors)
+[![Commits](https://img.shields.io/github/commit-activity/m/ampproject/amphtml.svg?logo=GitHub&style=flat-square "Commits")](https://github.com/ampproject/amphtml/pulse/monthly)
+
+[![GitHub Issues (open)](https://img.shields.io/github/issues/ampproject/amphtml.svg?logo=GitHub&style=flat-square "GitHub Issues (open)")](https://github.com/ampproject/amphtml/issues?q=is%3Aopen+is%3Aissue)
+[![GitHub Issues (closed)](https://img.shields.io/github/issues-closed/ampproject/amphtml.svg?logo=GitHub&style=flat-square "GitHub Issues (closed)")](https://github.com/ampproject/amphtml/issues?q=is%3Aissue+is%3Aclosed)
+[![GitHub Pull Requests (open)](https://img.shields.io/github/issues-pr/ampproject/amphtml.svg?logo=GitHub&style=flat-square "GitHub Pull Requests (open)")](https://github.com/ampproject/amphtml/pulls?q=is%3Aopen+is%3Apr)
+[![GitHub Pull Requests (closed)](https://img.shields.io/github/issues-pr-closed/ampproject/amphtml.svg?logo=GitHub&style=flat-square "GitHub Pull Requests (closed)")](https://github.com/ampproject/amphtml/pulls?q=is%3Apr+is%3Aclosed)
+
+##### Metrics:
+
+[![Absolute Code Coverage](https://img.shields.io/endpoint.svg?logo=Codecov&logoColor=white&style=flat-square&url=https%3A%2F%2Famp-project-metrics.appspot.com%2Fapi%2Fbadge%2FAbsoluteCoverageMetric "Absolute Code Coverage")](https://codecov.io/gh/ampproject/amphtml/)
+[![Presubmit Latency](https://img.shields.io/endpoint.svg?logo=Travis%20CI&logoColor=white&style=flat-square&url=https%3A%2F%2Famp-project-metrics.appspot.com%2Fapi%2Fbadge%2FPresubmitLatencyMetric "Presubmit Latency")](https://travis-ci.org/ampproject/amphtml/builds)
+[![Travis Greenness](https://img.shields.io/endpoint.svg?logo=Travis%20CI&logoColor=white&style=flat-square&url=https%3A%2F%2Famp-project-metrics.appspot.com%2Fapi%2Fbadge%2FTravisGreennessMetric "Travis Greenness")](https://travis-ci.org/ampproject/amphtml/builds)
```

Comment:
It's weird to see a greenness metric of 60% link to a page with 100% green builds (as is the case right now). Does the metric include PR builds too? (I'd assume not.) How far back do we go to calculate the percentage? Can we restrict this to a moving window of `master` builds over the past, say, 24 hours? I think this will offer a more accurate picture of the current level of Travis greenness.

/cc @danielrozenberg for your thoughts.

Reply:
This is based on the metric in the doc which was percentage of passing builds over a 90 day window. These are all pull_request builds.

Reply:
Okay, will continue this discussion in the doc. :+1:

Reply:
@rsimha Wouldn't we _want_ pull request builds? Since those are the ones merged with master?

Reply:
The builds for commits merged to `master` are of type `push`. See `TRAVIS_EVENT_TYPE` in [this](https://docs.travis-ci.com/user/environment-variables/#default-environment-variables) article.

Will discuss this offline.